FWCore/Integration/test/test_finalpath.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+LOCAL_TEST_DIR=${CMSSW_BASE}/src/FWCore/Integration/test
+LOCAL_TMP_DIR=${CMSSW_BASE}/tmp/${SCRAM_ARCH}
+
+# Pass in name and status
+function die { echo $1: status $2 ; echo === Log file === ; cat ${3:-/dev/null} ; echo === End log file === ; exit $2; }
+
+pushd ${LOCAL_TMP_DIR}
+
+cat <<EOF > finalpath_expected_empty.log
+EOF
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py >& finalpath.log || die "failed test_finalpath_cfg.py" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_empty.log - || die "differences for test_finalpath_cfg.py" $?
+
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --schedule >& finalpath.log || die "failed test_finalpath_cfg.py --schedule" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_empty.log - || die "differences for test_finalpath_cfg.py" $?
+
+
+cat <<EOF > finalpath_expected_not_found.log
+did not find thing '' TEST
+did not find thing '' TEST
+did not find thing '' TEST
+found thing 'beginLumi' TEST
+found thing 'endLumi' TEST
+found thing 'beginRun' TEST
+found thing 'endRun' TEST
+EOF
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --schedule --task >& finalpath.log || die "failed test_finalpath_cfg.py --schedule --task" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_not_found.log - || die "differences for test_finalpath_cfg.py --schedule --task" $?
+
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --endpath >& finalpath.log || die "failed test_finalpath_cfg.py --endpath" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_empty.log - || die "differences for test_finalpath_cfg.py --endpath" $?
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --endpath >& finalpath.log || die "failed test_finalpath_cfg.py --schedule --endpath" $?
```

Comment:
Should this have
```suggestion
cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --schedule --endpath >& finalpath.log || die "failed test_finalpath_cfg.py --schedule --endpath" $?
```
?

Reply:
fixed